deploy/ansible/roles/iroha-docker/templates/config.docker.j2

```diff
@@ -7,4 +7,13 @@
   "proposal_delay" : {{ iroha_proposal_delay }},
   "vote_delay" : {{ iroha_vote_delay }},
   "mst_enable" : {{ iroha_mst_enable }}
+  {% if iroha_mst_expiration_time is defined %}
```

Comment:
These new variables should also be defined in `defaults/main.yml`

Reply:
Since they are optional and have defaults values hardcoded, maybe I can put commented out examples in `main.yml`?

Reply:
Yes, this would work